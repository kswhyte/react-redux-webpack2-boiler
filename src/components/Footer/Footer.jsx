import React from 'react';
import PropTypes from 'prop-types';

const Footer = (props) => {

  Footer.propTypes = {
    title : PropTypes.string
  };

  const check = () => {
    console.log(props)
  }
  check();

    return (
        <footer className="footer-wrapper">
            <h1>Footer</h1>
        </footer>
    );
};

export default Footer;
