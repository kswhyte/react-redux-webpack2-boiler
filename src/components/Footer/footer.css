/* REFER to : /imports/layout.css for layout for footer */
