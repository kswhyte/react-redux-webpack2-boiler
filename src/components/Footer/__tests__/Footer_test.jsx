import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import Footer from '../Footer';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('Footer', module);

stories.add('Story', () => {
    const footerStory = (
      <Footer/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the Footer component without crashing', () => {
            let output = shallow(footerStory);
            expect(output.find('.footer-wrapper').length).toEqual(1);
        });
    }));
    return footerStory;
});
