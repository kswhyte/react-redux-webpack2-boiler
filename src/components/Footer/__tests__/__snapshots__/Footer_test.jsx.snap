// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Story 1`] = `
<div
  className="footer-wrapper"
>
  <h1>
    Footer
  </h1>
</div>
`;
